lib/ansible/module_utils/digital_ocean.py

```diff
@@ -96,6 +104,20 @@ def get_do_oauth_token(self):
             self.module.params.get('api_token') or \
             os.environ.get('DO_API_TOKEN') or \
             os.environ.get('DO_API_KEY') or \
-            os.environ.get('OAUTH_TOKEN')
+            os.environ.get('OAUTH_TOKEN') or \
+            os.environ.get('DO_OAUTH_TOKEN')
```

Comment:
Is this necessary if we use are using **digital_ocean_argument_spec**?

Reply:
ok.